src/client/datascience/jupyter/jupyterSession.ts

```diff
@@ -246,4 +266,37 @@ export class JupyterSession extends BaseJupyterSession {
             this.outputChannel.appendLine(output);
         }
     }
+
+    private async installDependenciesIntoInterpreter(
+        interpreter: PythonEnvironment,
+        cancelToken?: CancellationToken,
+        disableUI?: boolean
+    ) {
+        // TODO: On next submission move this code into a common location.
```

Comment:
This code is common between the old dependency checker, the kernel launcher, and the jupyter session. When I remove it from the kernel provider, I'll move it to a common location (probably the kernelDependencyService)

Reply:
That will happen in the next submit.